cli/command/container/client_test.go

```diff
@@ -32,14 +32,22 @@ func mockContainerLogsResult(content string) client.ContainerLogsResult {
 	return out
 }
 
+type fakeStreamResult struct {
+	io.ReadCloser
+	client.ImagePushResponse // same interface as [client.ImagePullResponse]
```

Comment:
Oh, just noticed `client.ImagePushResponse` doesn't follow the "XXXResult" pattern (`Response` vs `Result`)